### ISSUE TITLE
Can't open the editor

### ISSUE THREAD

**Issue by EmilEnglund** [defold/editor2-issues#318]
When trying to open the editor, windows thinks for 0.2 seconds then nothing happens. I get no error messages and can find no log to give you.

I'm on windows 10, 64-bit and trying to run the 64-bit windows version of Defold. Tried the 32-bit version of Defold with the same result.

I have disabled windows defender but it doesn't seem to make any difference.

**Comment by RagnarSvenssonKing** [comment]
Do you get a log-file here: C:\Users\<username>\AppData\Roaming\defold\editor2.log

**Comment by erikangelin-king** [comment]
Did you start Defold from the command line or explorer? Thanks for trying it out btw!

**Comment by EmilEnglund** [comment]
Trying to run the exe in the explorer. No log in my appdata. All that's there is a file named _crash and cache folders.

Looking forward to getting this working :)

**Comment by erikangelin-king** [comment]
We'll discuss this tomorrow!

**Comment by erikangelin-king** [comment]
Can’t reproduce this locally, can you help me out?

What is the full path of your Defold installation?

It is possible to start Defold from the command line if you give the full path to Defold.exe, like so:
C:\tmp\Defold-x86_64-win32\Defold> C:\tmp\Defold-x86_64-win32\Defold\Defold.exe
Can you try this? Please report the result of echo %ERRORLEVEL%

The Defold.exe just reads some config params and launches “java -jar …”. Can you try running:
C:\tmp\Defold-x86_64-win32\Defold\packages> .\jre\bin\java.exe -jar defold-2fd8732d146f5a252125ce1b210e3793f50e169c.jar
… with a suitable sha1 substituted in the jar. This should launch the editor splash screen at least, but the editor is probably not functional. With any luck, you will find an editor2.log in the packages directory.

Thanks!

**Comment by EmilEnglund** [comment]
The command line worked! Alright, so when using the command line, the editor opens without any problems. Double-clicking the .exe still does nothing.

I don't know what a sha1 is or how to substitute it, but I didn't get that command working. I'm using the same path as when I'm trying to start the editor but it tells me that it has an incorrect path.

**This works**
C:\program\Defold\Defold.exe

**This gives me "cannot find path"**
C:\program\Defold\packages> .\jre\bin\java.exe -jar defold-2fd8732d146f5a252125ce1b210e3793f50e169c.jar

**Comment by erikangelin-king** [comment]
Great you can at least get it running. Still can't reproduce but we might have a fix, will report back when it's available.

**Comment by erikangelin-king** [comment]
Hi! Can you please contact me at erik.angelin@king.com? I'd like to offer you an exclusive once-in-a-lifetime personalised Defold.exe.

**Comment by RagnarSvenssonKing** [comment]
By running it, you will be able to receive a large amount of money straight into your bank account. It's the top prize in a lottery you just won. Please accept before the prize goes to someone else.

**Comment by erikangelin-king** [comment]
I've shared a modified Defold.exe here:
https://drive.google.com/file/d/0BxFxQdv6jzsebUtYcjk3ZzY2UWs/view?usp=sharing

It would really help if you could try starting by double clicking, and also from command line - without giving the full path. Just replace the old Defold.exe.

Thanks!

**Comment by EmilEnglund** [comment]
Wow, this sounds amazing! Where can I send my account and credit card details?

Also, on my work comp that is running windows 10 enterprise N, Defold starts without any problems. Going to try on my home computer again with the new .exe later tonight.

**Comment by EmilEnglund** [comment]
The new .exe seems to give the same results. Double-click does nothing but I can start it through the command prompt. Tried "run as administrator", tried placing the Defold folder on a non-system drive and running it from there, tried updating java to the latest version but still no different results.

**Comment by erikangelin-king** [comment]
Hmm... do you get a console window with any more details when starting? Did you mean you could start it without the full path? Thanks a lot for testing! So frustrating when you can't reproduce issues :)

**Comment by EmilEnglund** [comment]
No console window.

I have to input the path to the exe to start the editor, like this:
C:\program\Defold\Defold.exe
If I go into the Defold directory and input "Defold.exe" it does not work.
Actually, inputting "Defold.exe" in the Defold directory doesn't work on my work computer either.

No worries, I'm curious about this issue too :)

**Comment by erikangelin-king** [comment]
And you're sure you replaced the installed defold.exe with the one i shared?

**Comment by EmilEnglund** [comment]
At home where nothing is working, I had replaced the .exe with the one you sent, still not working.

At work - where the only thing that wasn't working was starting defold through the command prompt without the full path - I had not. When I replaced the .exe starting through the prompt without the full path started working, with this output:

`DEBUG:TOOLS: arg 0: C:/Program Files/Defold/packages/jre/bin/java
DEBUG:TOOLS: arg 1: -cp
DEBUG:TOOLS: arg 2: C:/Program Files/Defold/packages/defold-2fd8732d146f5a252125ce1b210e3793f50e169c.jar
DEBUG:TOOLS: arg 3: -Ddefold.editor2.log=C:\Users\emil.englund\AppData\Roaming\Defold/editor2.log
DEBUG:TOOLS: arg 4: -Ddefold.resourcespath=C:/Program Files/Defold
DEBUG:TOOLS: arg 5: -Ddefold.version=1.2.95
DEBUG:TOOLS: arg 6: -Ddefold.sha1=2fd8732d146f5a252125ce1b210e3793f50e169c
DEBUG:TOOLS: arg 7: -Djava.ext.dirs=C:/Program Files/Defold/packages/jre/lib/ext
DEBUG:TOOLS: arg 8: -Djava.net.preferIPv4Stack=true
DEBUG:TOOLS: arg 9: -Dsun.net.client.defaultConnectTimeout=30000
DEBUG:TOOLS: arg 10: -Dsun.net.client.defaultReadTimeout=30000
DEBUG:TOOLS: arg 11: -Djogl.texture.notexrect=true
DEBUG:TOOLS: arg 12: com.defold.editor.Start
DEBUG:TOOLS: arg 13: (null)
DEBUG:TOOLS: "C:/Program Files/Defold/packages/jre/bin/java" -cp "C:/Program Files/Defold/packages/defold-2fd8732d146f5a252125ce1b210e3793f50e169c.jar" -Ddefold.editor2.log=C:\Users\emil.englund\AppData\Roaming\Defold/editor2.log "-Ddefold.resourcespath=C:/Program Files/Defold" -Ddefold.version=1.2.95 -Ddefold.sha1=2fd8732d146f5a252125ce1b210e3793f50e169c "-Djava.ext.dirs=C:/Program Files/Defold/packages/jre/lib/ext" -Djava.net.preferIPv4Stack=true -Dsun.net.client.defaultConnectTimeout=30000 -Dsun.net.client.defaultReadTimeout=30000 -Djogl.texture.notexrect=true com.defold.editor.Start`

**Comment by EmilEnglund** [comment]
I found the problem! And it was a windows-one...

I had UAC (Windows User Account Control) turned off, and one might think that turning it off would allow me to start whichever app I wanted without windows interfering - turns out that it still did prevent the app from starting, only, without telling me.

I reactivated UAC and it then gave me a message saying that Smart Screen had prevented the app from starting. Disabling Smart Screen fixed it so now I can start it through the explorer without any problems.
